src/fmcore/prompt_tuner/base_prompt_tuner.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+from abc import ABC, abstractmethod
+from typing import Dict
+
+from fmcore.prompt_tuner.types.prompt_tuner_types import PromptTunerConfig, PromptTunerResult
+from fmcore.types.enums.dataset_enums import DatasetType
+from fmcore.types.typed import MutableTyped
+from bears.util import Registry
+
+
+class BasePromptTuner(MutableTyped, Registry, ABC):
```

Comment:
TODO to replace this as one interface of each prompt tuner and not have runners and tuners and multiple interfaces

Reply:
With and ETA of today